perf-regression/theta-gpu/README.md

```diff
@@ -1,4 +1,25 @@
-This is an example script for executing an automated regression test on the
+To run the margo gpu test: gpu-margo-p2p-bw.cu with the current libfabric release, 
```

Comment:
I think rather than explaining the modifications a user would need to do, we should instead just make a PR to the mochi-spack-packages repo with the changes needed and then just describe that libfabric needs to be configured with the --with-cuda variant.

Reply:
I will add the changes described here to llibfabric's package.py and create a PR at https://github.com/mochi-hpc/mochi-spack-packages.